### PR TITLE
Update string ref to use createRef

### PR DESCRIPTION
**Pull request by ajbowler**
I'm not sure if I need to commit the `dist/` directory, let me know if I need to remove that.

Closes #13

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -54,7 +56,7 @@ export default class TwentyTwenty extends Component {
         return;
       }
     }
-    
+
     if (isTouch) e.preventDefault();
 
     const { left, width } = this.refs.component.getBoundingClientRect();
```
Comment (jacobp100) [inline src/index.js]: We need `.current` here

Comment (ajbowler) [reply src/index.js]: Right you are, updating

**Comment by jacobp100** [comment]
Thanks! Just one small fix.

I was committing the `dist` directory so the demo works.

File: src/index.js (hunk 1 above)
Comment (ajbowler) [reply src/index.js]: Done!

**Comment by jacobp100** [comment]
Thanks!

**Comment by ajbowler** [comment]
Glad to help! Could we get a release for this?

**Comment by jacobp100** [comment]
v1.2.0 is released!

**Pull request merged**